packages/sit-onyx/src/components/OnyxRadioButton/OnyxRadioButton.vue

```diff
@@ -153,6 +155,7 @@ const skeleton = useSkeletonContext(props);
       background-color: var(--onyx-radio-button-selector-background-color);
       outline: var(--onyx-radio-button-selector-outline-width) solid
         var(--onyx-radio-button-selector-outline-color);
+      outline-offset: 0;
```

Comment:
Why is this needed?

Reply:
I don't now why but there was an 2px outline-offset.